github_client/github.go

```diff
@@ -1,11 +1,12 @@
-package github
+package github_client
```

Comment:
Potential Bug: Make sure that importing "github.com/birdsean/review-droid/comments" is necessary and is being used properly, or remove the import statement.